db/seeds.rb

```diff
@@ -712,6 +712,14 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   )
 end
 
+class Turbo::StreamsChannel
+  [:broadcast_append_to, :broadcast_prepend_to, :broadcast_replace_to, :broadcast_remove_to].each do |method|
+    define_singleton_method(method) do |*args|
+      return nil
+    end
+  end
+end
```

Comment:
Why is this in `db:seeds`?

Reply:
@dorner I think it is to avoid making those callbacks trigger turbo broadcasts that are defined on the model. Lemme double check if its actually needed.

Reply:
Removing, I don't recall why it was added. I don't think we have any models that broadcast from a callback